Dockerfile-0

```diff
@@ -0,0 +1,3 @@
+FROM ubuntu
+RUN apt-get update
+RUN apt-get install -y g++ gdb make
```

Comment:
Why two docker files... we can combine them into one... docker should cache the steps anyways... just make sure to order them correctly.